sycl/include/CL/__spirv/spirv_ops.hpp

```diff
@@ -75,8 +74,8 @@ template <typename T, typename Tp, std::size_t R, std::size_t C,
           __spv::Scope::Flag S = __spv::Scope::Flag::Subgroup>
 extern __DPCPP_SYCL_EXTERNAL void __spirv_JointMatrixStoreCheckedINTEL(
     T *Ptr, __spv::__spirv_JointMatrixINTEL<Tp, R, C, L, S, U> *Object,
-    std::size_t Stride, size_t Height, size_t Width, size_t CoordX,
-    size_t CoordY, __spv::MatrixLayout Layout = L, __spv::Scope::Flag Sc = S,
+    std::size_t Stride, int32_t Height, int32_t Width, int32_t CoordX,
+    int32_t CoordY, __spv::MatrixLayout Layout = L, __spv::Scope::Flag Sc = S,
     int MemOperand = 0);
```

Comment:
I also prefer using uint32_t per https://github.com/intel-restricted/applications.compilers.llvm-project/pull/20017#discussion_r1562141264 comment

Reply:
To double check (as SPIR-V spec doesn't (and actually can't specify that)), can't CoordX and CoordY be signed?

Reply:
As a matter of fact, the coordinates can be negative numbers. Thanks for checking.